feature-libs/pickup-in-store/components/presentational/pickup-options/pickup-options.component.html

```diff
@@ -3,7 +3,7 @@
     <input
       type="radio"
       role="radio"
-      id="delivery"
+      data-pickup="delivery"
```

Comment:
We need the `id` attribute to associate the label with it

Reply:
Added [attr.id]